<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QInputTransformDialog</class>
 <widget class="QDialog" name="QInputTransformDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDoubleSpinBox" name="x_val_spinbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999999.990000000223517</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="y_val_spinbox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999999.990000000223517</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="z_val_spinbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999999.990000000223517</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rx_val_spinbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99999999.989999994635582</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ry_val_spinbox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99999999.989999994635582</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rz_val_spinbox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999999.990000000223517</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x            y           z   </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rx          ry           rz   </string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>x_val_spinbox</zorder>
  <zorder>y_val_spinbox</zorder>
  <zorder>z_val_spinbox</zorder>
  <zorder>rx_val_spinbox</zorder>
  <zorder>ry_val_spinbox</zorder>
  <zorder>rz_val_spinbox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
